code/modules/surgery/organs/ears.dm

```diff
@@ -166,6 +170,78 @@
 		ear_owner.dna.species.mutant_bodyparts -= "ears"
 		ear_owner.update_body()
 
+/obj/item/organ/ears/rabbit
+	name = "rabbit ears"
+	damage_multiplier = 3
```

Comment:
this can just be the same as the others to make it not a straight downgrade

Reply:
Given the comically large size of the ears and how rabbits have substantially better hearing than cats, dogs, and foxes, I thought it was appropriate. I'll patch it back down to two though!